client/src/components/App/Content/LandingPage/LandingPageStyles.ts

```diff
@@ -0,0 +1,18 @@
+import { makeStyles } from '@material-ui/styles';
+
+import theme from 'styles/theme';
+
+const useStyles = makeStyles({
+    content: {
+        height: '84vh',
+        backgroundColor: '#F3F6FB',
+    },
+    welcomeMessage: {
+        fontSize: '4vh',
+        color: theme.palette.text.primary,
```

Comment:
Maybe typography has a color prop and you can send there 'primary' instead of using it here